docs/intro.rst

```diff
@@ -41,34 +41,36 @@ Key Features
 Quickstart
 ----------
 
-Assuming you have already installed Confluent Platform. You'll need to specify the location
-of Camus's jar file when running the MapReduce job. If you install Camus via zip/tgz archive,
-you can find Camus's jar under ``share/java/camus/``. If you install Camus via rpm or deb,
-the Camus's jar is under ``/usr/share/java/camus/``.
-
 .. ifconfig:: platform_docs
 
-   Also, you need to have access to a Hadoop Cluster. For installation and deployment of a single node
+   These instructions assume you have already installed Confluent Platform and that
+   you have access to a Hadoop Cluster. For installation and deployment of a single node
    Hadoop in pseudo-distributed mode, see this
    `guide <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cdh_qs_cdh5_pseudo.html>`_.
    Finally, you should have :ref:`Kafka and the Schema Registry running<quickstart>` .
 
 .. ifconfig:: not platform_docs
 
-   Also, you need to have access to a Hadoop Cluster. For installation and deployment of a single node
+   These instructions assume you have already installed Confluent Platform and that
```

Comment:
This is intentional -- the two blocks are used in different cases (when building the complete platform docs vs. just docs for this repository). The `ifconfig` command only works at the paragraph level, so we have a small bit of duplication between them.